### DISCUSSION TITLE
A fork of Remix's Epic Stack that uses Ariakit

### DISCUSSION THREAD

**Discussion by gnapse** [ariakit/ariakit#3583] [General]
This is not a question, but just sharing something I made.

I'm intending to play with Remix using its [Epic Stack](https://www.epicweb.dev/epic-stack), but it comes with [Radix UI](https://www.radix-ui.com) out of the box. So I decided to make a fork that switches over to Ariakit instead.

Check it out at https://github.com/gnapse/epic-stack

**Comment by diegohaz** [comment]
That's so cool! Thanks for sharing.
